wordpress-mobile/WordPress-iOS
# Migrates PlanFeatureServiceRemote and PlanServiceRemote to WordPressKit

Fixes #7319 and #7318 

This PR:
* Moves `String+Helpers.swift` and `StringHelperTests.swift` to the WordPressShared framework
* Migrates the Plan model to the WPKit framework
* Migrates `PlanFeatureServiceRemote` to the WPKit framework
* Migrates `PlanServiceRemote` to the WPKit framework
* Migrates the appropriate remote tests to the WPKit framework

Ref #7301

**To test:**
1. Clean DerivedData.
2. Update CocoaPods repo with pod repo update.
3. Install the pods with pod install.
4. Run _all_ unit tests and make sure they are green.
5. Build & run the app and verify the Plans screens are working as expected.

Needs review: @astralbodies
